ui/helpers/utils/util.js

```diff
@@ -443,3 +443,77 @@ export const toHumanReadableTime = (t, milliseconds) => {
 export function clearClipboard() {
   window.navigator.clipboard.writeText('');
 }
+
+const solidityTypes = () => {
+  const types = [
+    'bool',
+    'address',
+    'string',
+    'bytes',
+    'int',
+    'uint',
+    'fixed',
+    'ufixed',
```

Comment:
We should also include the explicitly sized versions of `fixed` and `ufixed`, i.e. `ufixedMxN` where M is divisible by 8 and between 8 and 256 (inclusive), and N is an integer between 0 and 80.

Details: https://docs.soliditylang.org/en/latest/types.html#fixed-point-numbers